src/core/credGrainView.js

```diff
@@ -54,77 +60,133 @@ export type ParticipantsOptions = {|
 export class CredGrainView {
   _credGraph: CredGraph;
   _ledger: Ledger;
-  _defaultTimeScope: TimeScopedCredGrainView;
+  _participants: Array<ParticipantCredGrain>;
+  _intervals: IntervalSequence;
 
   constructor(credGraph: CredGraph, ledger: Ledger) {
     this._credGraph = credGraph;
     this._ledger = ledger;
-    this._defaultTimeScope = new TimeScopedCredGrainView(
-      credGraph,
-      ledger,
-      -Infinity,
-      Infinity
-    );
+    this._participants = [];
+    this._intervals = credGraph.intervals();
+
+    const graphParticipants = new Map<IdentityId, GraphParticipant>();
+    for (const participant of credGraph.participants()) {
+      graphParticipants.set(participant.id, participant);
+    }
+    for (const account of ledger.accounts()) {
+      const graphParticipant = graphParticipants.get(account.identity.id);
+      if (!graphParticipant)
+        throw new Error(
+          "The graph is missing an account that exists in the ledger."
```

Comment:
would be helpful to include the id and identity name for the missing identity.

Reply:
accepted